### PR TITLE
etcdctl: make doc on 'no-sync' clearer

### PR DESCRIPTION
**Pull request by gyuho**
For issues like https://github.com/coreos/etcd/issues/4496.

### CODE REVIEW

File: etcdctl/README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ You can also build etcdctl from source using the build script found in the paren
 
 ### --no-sync
 + don't synchronize cluster information before sending request
++ Use this to access non-published client endpoints
++ Without this flag, values from `--endpoint` flag will be overwritten by the values from peer API
```
Comment (xiang90) [inline etcdctl/README.md]: will be overwritten by etcd cluster when it does internal sync.

Comment (gyuho) [inline etcdctl/README.md]: Ok will fix. Thanks.

**Comment by gyuho** [comment]
@xiang90 Just addressed. PTAL. Thanks.

**Comment by xiang90** [comment]
LGTM

**Pull request merged**